File: Python/NumberConversion/Bulid-in/decimal_to_all.py
# decimal_conversion_builtin.py

num=int(input("Enter a decimal number: "))

binary=bin(num)[2:]
octal=oct(num)[2:]
hexadecimal=hex(num)[2:].upper()

print("\n--- Conversion Result ---")
print("Binary      =",binary)
print("Octal       =",octal)
print("Hexadecimal =",hexadecimal)

File: Java/NumberConversion/NoBulid-in/Decimal_to_all.java
import java.util.*;

public class Decimal_to_all {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter decimal number: ");
        int num = sc.nextInt();

        String binary = "";

        if (num == 0) {
            binary = "0";
        } else {
            while (num > 0) {
                int rem = num % 2;
                binary = rem + binary;
                num = num / 2;
            }
        }

        System.out.println("Binary = " + binary);
    }
}
